e2e/framework/testserver.go

```diff
@@ -38,7 +39,7 @@ type TestServerConfigCallback func(*TestServerConfig)
 
 const (
 	initialPort   = 12000
-	polygonSDKCmd = "polygon-sdk"
+	polygonSDKCmd = "polygon-sdk-race"
```

Comment:
This is a problem.

We can't just overwrite the command we use for the testing server and consider that every local build is going to have a race safe binary.

This is in relation to the step where we build out the binary the test server will use:
https://github.com/0xPolygon/polygon-sdk/blob/test/build_with_race_flag/e2e/README.md#step-1-build-the-polygon-sdk

I'd like to discuss this @munna0908 @lazartravica @Kourin1996